.dialyzer.ignore-warnings

```diff
@@ -1,3 +1,75 @@
+-------------------------------
+ # TODO:: EX WIRE IS GOING THROUGH LARGE REFACTOR
```

Comment:
@hayesgm I added ex_wire here

Reply:
maybe we can leave `ex_wire` ignored? because we'll have to update dialyzer ignore file after every change if  this gets merged

Reply:
afaik, what @hayesgm is doing is going to change much of typespecs in ex_wire. until then, I would prefer if we strictly don't add any more issues to that pile. if we add the whole app then we're ignoring potential increases.

Reply:
when @hayesgm rebases, he'll need to clean the ignore file though.